test/executor.test.js

```diff
@@ -809,6 +882,78 @@ describe('executor', function() {
     });
   });
 
+  describe('dynamic configuration for datasources.json', function() {
+
+    afterEach(function() {
+      delete process.env.DYNAMIC_HOST;
+      delete process.env.DYNAMIC_PORT;
+    });
```

Comment:
In general, it's better to clean up before the test than after the test. That way when the test is over, you can still inspect the environment in which it was running. Although I admit this is more useful when it comes to files & databases than here with env variables.